server.coffee

```diff
@@ -7,7 +7,7 @@ application = module.exports = (options, callback) ->
     options.name = 'Kresus'
     options.root ?= __dirname
     options.port ?= process.env.PORT || 9876
-    options.host = process.env.HOST || "127.0.0.1"
+    options.host = process.env.IP || "127.0.0.1"
```

Comment:
I had an issue with this line while setting up my online IDE and the change got commited by error. Sorry!